source/parser.apib

```diff
@@ -138,7 +138,7 @@ General-purpose result of the parsing operation. The parse result is in form of
 
     + Headers
 
-            Accept: application/raml+yaml
+            Content-Type: application/raml+yaml
```

Comment:
You can remove this block then and put the content type on line 137.

Reply:
Actually, don't remove the block, but keep a `Accept: application/vnd.refract.parse-result+yaml` here.

Reply:
Can you please add some justification why the Accept header should be present in this requst?

I don't think it should be present because the example is to demonstrate behaviour with the content in the request. It adds noise and subtracts from the example making it less clear what the example is trying to demonstrate.

Reply:
I don't mind the state of accept here. Mistaknely not phrased my second comment as a suggestion